test/c3/appframework/appframework_test.go

```diff
@@ -2616,4 +2616,296 @@ var _ = Describe("c3appfw test", func() {
 
 		})
 	})
+
+	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
+		It("integration, c3, appframework: can deploy a C3 SVA and have apps installed abd updated locally on Cluster Manager and Deployer for manual polling", func() {
+
+			/* Test Steps
+			   ################## SETUP ####################
+			   * Upload V1 apps to S3
+			   * Create app source with local scope for C3 SVA (Cluster Manager and Deployer)
+			   * Prepare and deploy C3 CRD with app framework and wait for pods to be ready
+			   ############# INITIAL VERIFICATION ##########
+			   * Verify Apps are Downloaded in App Deployment Info
```

Comment:
Refactoring needed to identify common steps and moved into common methods. CSPL-1580 created